### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bank1e**
Drafted installation guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,40 @@ Meeting Agenda templates (located in the `/docs` directory in this repo):
...
+```
+npm install
+```
+#### 3. Create .env file both in "client" and "backend" directory.
+#### client > .env
+```
+# Demo User Credentials
+VITE_DEMO_USERNAME=demo_user
+VITE_DEMO_PASSWORD=demo4929
+VITE_BACKENDURL=https://v49-tier3-team-29.onrender.com
```
Comment (jaredsina) [inline README.md]: The VITE_BACKENDURL should be set to their localhost backend

File: README.md [hunk 2]
```diff
@@ -27,6 +27,40 @@ Meeting Agenda templates (located in the `/docs` directory in this repo):
...
+#### 2. Install npm packages both in "client" and "backend" directory.
+```
+npm install
+```
+#### 3. Create .env file both in "client" and "backend" directory.
+#### client > .env
+```
+# Demo User Credentials
+VITE_DEMO_USERNAME=demo_user
+VITE_DEMO_PASSWORD=demo4929
```
Comment (jaredsina) [inline README.md:46]: Would VITE_DEMO_PASSWORD have to match up with the DEMOUSER_PASSWORD on the backend?

Comment (bank1e) [reply README.md:46]: Yes, they should be identical. Thanks for reminding me. I will add the information.

**Review by jaredsina — APPROVED** [review]

**Review by jaredsina — APPROVED** [review]
Thank you for the updates Bia! Great job on this.

**Pull request merged**